apps/vscode/vscode.talon

```diff
@@ -14,6 +14,10 @@ please [<user.text>]:
     user.vscode("workbench.action.showCommands")
     insert(user.text or "")
 
+view [<user.text>]:
```

Comment:
Would this make more sense as a .talon-list, rather than user.text? Seems like it would be more reliable, with the downside of needing to add an entry for each view.

Reply:
I prefer the test option because I'm using it to discover views I doing know about. It's kind of like "please" for searching views. If I had to delineate all the options, it would be less useful to me. There are several commands already for toggling bars and panels and such but there isn't such a convenient view search option yet (imo)

Reply:
I'm okay with this, but I think we should prefix the command with "go"

Reply:
I'm fine with that.